jobs/delorean/keycloak/next/branch.yaml

```diff
@@ -0,0 +1,28 @@
+---
+- job:
+    name: delorean-keycloak/next/branch
+    display-name: 'keycloak-next-branch'
+    project-type: pipeline
+    concurrent: false
+    parameters:
+      - string:
+          name: 'installationGitUrl'
```

Comment:
Sure, I use it like this so there wouldn't be lots of changes in the jenkinsfile or as little changes as possible

Reply:
It might have a knock on effect if we change it now. Probably best to leave it, or consider moving to a different Jenkinsfile if we think it's different enough, maybe it isn't.